src/main/java/com/raisetech/mybatisdemo/UserServiceImpl.java

```diff
@@ -8,43 +8,58 @@
 
 @Service
 public class UserServiceImpl implements UserService {
-    private final UserMapper nameMapper;
+    private final UserMapper userMapper;
 
-    public UserServiceImpl(UserMapper nameMapper) {
-        this.nameMapper = nameMapper;
+    public UserServiceImpl(UserMapper userMapper) {
+        this.userMapper = userMapper;
     }
 
     @Override
     public List<User> findAll() {
-        return nameMapper.findAll();
+        return userMapper.findAll();
     }
 
     @Override
     public Optional<User> findById(int id) {
-        Optional<User> name = this.nameMapper.findById(id);
-        return Optional.ofNullable(nameMapper.findById(id).orElseThrow(() -> new ResourceNotFoundException("resource not found")));
+        return Optional.ofNullable(userMapper.findById(id).orElseThrow(() -> new ResourceNotFoundException("resource not found")));
```

Comment:
userMapperのfindByIdがOptionalを返すので下記のようにすれば良いはずです。
```suggestion
        return userMapper.findById(id).orElseThrow(() -> new ResourceNotFoundException("resource not found"));
```

Reply:
修正しました！これで大丈夫でしょうか？
c27be66a66fd1e3734e058dea45e17556bd60d7a

Reply:
問題ないです！